templates/named.conf.options.erb

```diff
@@ -127,3 +127,25 @@ statistics-channels {
     inet <%= @statistic_channel_ip %> port <%= @statistic_channel_port %><% if @statistic_channel_allow -%> allow { <%= @statistic_channel_allow.join('; ') %>; }<% end -%>;
 };
 <% end -%>
+
+<% if @query_log_enable -%>
+logging {
+
+        channel query_logging {
+                file "/var/cache/bind/named_querylog"
```

Comment:
${data_dir} is on Debian systems '/etc/bind/zones', I wouldn't use this for log file targetting. A new variable is required for that.